src/Microsoft.Azure.SignalR.AspNet/Infrastructure/ServiceConnection.cs

```diff
@@ -92,14 +92,7 @@ protected override Task OnMessageAsync(ConnectionDataMessage connectionDataMessa
                     var payload = connectionDataMessage.Payload;
                     Log.WriteMessageToApplication(_logger, payload.Length, connectionDataMessage.ConnectionId);
                     var message = GetString(payload);
-                    if (message == ReconnectMessage)
-                    {
-                        transport.Reconnected?.Invoke();
-                    }
-                    else
-                    {
-                        transport.OnReceived(message);
-                    }
+                    transport.Output.WriteAsync(message);
```

Comment:
>message [](start = 48, length = 7)

write `payload` instead?

Reply:
`_ = ` ?

---
In reply to: [234902146](https://github.com/Azure/azure-signalr/pull/268#discussion_r234902146) [](ancestors = 234902146)

Reply:
how about write connectionDataMessage? And log can be moved inside ProcessOutgoingMessage().